vagga.yaml

```diff
@@ -0,0 +1,28 @@
+containers:
+    gdg_dev:
+      setup:
+      - !Ubuntu trusty
+      - !UbuntuUniverse
+      - !Install [software-properties-common, curl, git, openssl, libssl-dev, libpq-dev, ca-certificates, build-essential, wget, python-pip, python-dev, mercurial]
+      - !Sh |
+          curl -sL https://deb.nodesource.com/setup_5.x | sudo -E bash -
+          sudo add-apt-repository -y ppa:fkrull/deadsnakes
+          sudo apt-get update
+      - !Install [python3.5, nodejs]
+      - !Sh |
+          pip install virtualenv
+commands:
+  run: !Supervise
+    description: Run app - testing status
+    children:
+      env: !Command
+        container: gdg_dev
+        run: |
+          export OAUTHLIB_INSECURE_TRANSPORT=1
+          if [ ! -d "gdg.org.ua" ]; then
+          git clone https://github.com/GDG-Ukraine/gdg.org.ua
+          fi
+          cd gdg.org.ua
+          make env
```

Comment:
You may drop `make env`. It just create a virtualenv in `.env` folder. And then try running `sudo make dev-deps`, installing python packages globally. But there may be a side effect if `.env` folder already exists, so preferable way of installing packages is to either (1) run installation manually (as in doc), or introduce some variable to `Makefile` making possible to let it ignore activating virtualenv at all.